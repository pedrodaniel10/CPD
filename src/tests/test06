3 10 100 5