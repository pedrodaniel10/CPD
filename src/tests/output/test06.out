0.58 0.25
0.50 0.51
